econplayground/main/views.py

```diff
@@ -386,8 +386,29 @@ def dispatch(self, request, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         context = super(CohortListView, self).get_context_data(**kwargs)
+
+        # This area should never be reached if this user isn't an instructor,
+        # but just in case.
+        if not user_is_instructor(self.request.user):
+            return context
+
+        # Create a clone of the sample course if this user has no
+        # courses.
+        if Cohort.objects.filter(
+                instructors__in=(self.request.user,)).count() == 0:
```

Comment:
`exists` is a bit more efficient than `count`. (https://docs.djangoproject.com/en/2.2/ref/models/querysets/#django.db.models.query.QuerySet.exists)

Reply:
Oh yeah - Thanks! I forgot about that. It's been a while since I used it.